src/EmployerFeedback/SFA.DAS.EmployerFeedback/InnerApi/Requests/GetSettingsRequest.cs

```diff
@@ -0,0 +1,9 @@
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerFeedback.InnerApi.Requests
+{
+    public class GetSettingsRequest : IGetApiRequest
```

Comment:
This could do with it's name changing to make it more specific now that the setting is named